lib/locale_threads.t

```diff
@@ -11,6 +11,8 @@ BEGIN {
     set_up_inc('../lib');
 
     skip_all_without_config('useithreads');
+    skip_all("Fails on threaded builds on OpenBSD and Dragonfly")
```

Comment:
The styling is DragonFly with a capital "F"

https://en.wikipedia.org/wiki/DragonFly_BSD

Reply:
Agreed, per https://www.dragonflybsd.org/